cmd/package-operator-manager/main.go

```diff
@@ -71,6 +74,9 @@ func main() {
 	flag.StringVar(&opts.loadPackage, "load-package", "", "(internal) runs the package-loader sub-component to load a package mounted at /package")
 	flag.StringVar(&opts.selfBootstrap, "self-bootstrap", "",
 		"(internal) bootstraps Package Operator with Package Operator using the given Package Operator Package Image")
+	flag.StringVar(&opts.remotePhasePackageImage, "remote-phase-package-image", os.Getenv("PKO_REMOTE_PHASE_PACKAGE_IMAGE"),
+		"Providing an image will enabling the deployment of an additional controller that creates a remote phase manager package"+
+			"for every HostedCluster object")
```

Comment:
Either "Image to use for the remote phase manager package" or "Image to use for the remote phase manager package that is created for every HostedCluster object if HyperShift HostedCluster API is present on the cluster"